### DISCUSSION TITLE
allow grant to specific tables

### DISCUSSION THREAD

**Discussion by martingehrke** [cyrilgdn/terraform-provider-postgresql#83] [General]
Any plans to bring this over from the old project?
https://github.com/hashicorp/terraform-provider-postgresql/pull/156

**Comment by cyrilgdn** [comment]
Hi @martingehrke ,


FYI this is done in [v1.14.0](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest) with the new [`objects` argument in `postgresql_grant`](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs/resources/postgresql_grant#objects)
